eureka2-server/src/main/java/com/netflix/eureka2/server/channel/InterestNotificationMultiplexer.java

```diff
@@ -112,6 +112,6 @@ public void unregister() {
      * simplifies implementation of the multiplexer.
      */
     public Observable<ChangeNotification<InstanceInfo>> changeNotifications() {
-        return aggregatedStream;
+        return aggregatedStream.share();
```

Comment:
As part of this pull InterestChannel have a new API getChangeNotificationStream(). On the server side this is bridged to the Multiplexer's changeNotifications so even though we don't subscribe to this stream more than once, this share() is to guard against that per the new contract.